caffe2/CMakeLists.txt

```diff
@@ -1017,8 +1017,8 @@ endif()
 if(LINUX)
   find_library(BREAKPAD_LIB breakpad_client)
   find_path(BREAKPAD_INCLUDE_DIR breakpad)
-  if(BREAKPAD_LIB_FOUND AND BREAKPAD_INCLUDE_DIR_FOUND)
-    target_link_libraries(torch_cpu PRIVATE ${BREAKPAD_LIB})
+  if(BREAKPAD_LIB AND BREAKPAD_INCLUDE_DIR)
+    target_link_libraries(torch_cpu PUBLIC ${BREAKPAD_LIB})
```

Comment:
When this is `PRIVATE` instead of `PUBLIC` I get undefined symbol errors from `ld` for symbols in `libbreakpad_client.a` during loading (e.g. `import torch`). Is that because `libtorch_python.so` transitively depends on `libbreakpad_client`? I figured since its a static lib it shouldn't matter outside `libtorch_cpu.so` but that doesn't seem to be the case